### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by adrienemery**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/adrienemery/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+- The issue has been resolved.
+
+To add a flag simply leave a comment with your desired flag. For
+convenience we also have a dropdown menu to add these comments
+automatically.
+
+Another key feature you need to know about is ninja stars. Ninja 
+stars are how you communicate that a pull request is okay to merge. 
+You can set, by repository, how many ninja stars you think pull 
+requests should have before merging.
```
Comment (adrienemery) [inline ReadMe.md:29]: some comment here

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+Line notes have flags to indicate what is going on in the code,
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
+
+`['!fixed', '!resolved', '!completed']`
+- The issue has been resolved.
+
```
Comment (adrienemery) [inline ReadMe.md:21]: another comment here

File: ReadMe.md (hunk 1 above)
Comment (adrienemery) [inline ReadMe.md:29]: more comments

File: ReadMe.md (hunk 2 above)
Comment (adrienemery) [inline ReadMe.md:21]: !fix this line

File: ReadMe.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+What you’re seeing here is a GitHub pull request with the code diffs
+showing by default. Feel free to look around.
+
+Over on the right is a panel for line notes that you can connect to a
+line of code in the diff. You can also switch to the conversation
+to see comments on the whole pull request.
+
+Line notes have flags to indicate what is going on in the code,
+too. Flags are as follows:
```
Comment (adrienemery) [inline ReadMe.md:13]: !resolved

File: ReadMe.md (hunk 2 above)
Comment (adrienemery) [inline ReadMe.md:21]: !resolved

**Pull request merged**